CMakeLists.txt

```diff
@@ -513,13 +513,13 @@ if(ENABLE_CPACK)
                    ${CPACK_DEBIAN_PACKAGE_DEPENDS})
   endif()
 
-  # soft requirement of tcmalloc if selected
-  if(USE_TCMALLOC)
-    message(STATUS "Adding gperftools to the package requirements")
+  # soft requirement of jemalloc if selected
+  if(USE_JEMALLOC)
+    message(STATUS "Adding jemalloc to the package requirements")
     set(CPACK_RPM_PACKAGE_REQUIRES
-        "${CPACK_RPM_PACKAGE_REQUIRES},gperftools-libs >= 2.0")
```

Comment:
I only just spotted this but for these user package dependencies can we just use the non-development packages? It avoids installing any development dependencies on user machines as they include non-versioned symlinks that prevent multiple package versions being installed together.

For RPM this would be `jemalloc`.
For DEB this would be `libjemalloc1`.